src/base/net/downloadmanager.cpp

```diff
@@ -148,10 +148,7 @@ Net::DownloadManager::DownloadManager(QObject *parent)
         QStringList errorList;
         for (const QSslError &error : errors)
             errorList += error.errorString();
-        LogMsg(tr("Ignoring SSL error, URL: \"%1\", errors: \"%2\"").arg(reply->url().toString(), errorList.join(u". ")), Log::WARNING);
-
-        // Ignore all SSL errors
-        reply->ignoreSslErrors();
+        LogMsg(tr("SSL error, URL: \"%1\", errors: \"%2\"").arg(reply->url().toString(), errorList.join(u". ")), Log::CRITICAL);
```

Comment:
The `Log::CRITICAL` was reserved for errors that would cause qbt unable to continue operation, i.e. fatal errors. It was right to use `Log::WARNING`.